app/views/gettingStarted.scala.html

```diff
@@ -20,52 +20,57 @@ <h1>Getting Started with Play Framework</h1>
 
             <div class="try-section">
                 <h2>New to Play?</h2>
-                <p>Start with <a href="@controllers.documentation.ReverseRouter.latest(None)">our documentation</a> or the <a href="@controllers.documentation.ReverseRouter.latest(None, "HelloWorldTutorial")">Hello World Tutorial</a>.</p>
-
                 <div class="try-option">
                     <h3><span>&raquo;</span> Learn whether Play Framework is right for your project</h3>
                     <div class="try-option-content">
                         <ul>
-                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Introduction")">Introduction to Play</a></li>
-                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")">Requirements for using Play</a></li>
-                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Philosophy")">Our philosophy</a></li>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Introduction")" target="_blank">Introduction to Play</a></li>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Philosophy")" target="_blank">Our philosophy</a></li>
+                            <li><a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")" target="_blank">Requirements for using Play</a></li>
```

Comment:
Why `target="_blank"`?

Reply:
Hi @dwijnand, Because if the reader wants to go read about Play and come back and try the examples, this way the page will still be open in their original tab. I've been doing a lot of documentation research lately and this is a common best practice when the link is supplemental information but the reader likely wants to come back and continue with the page.

Reply:
OK, interesting :)